src/test/regress/expected/columnar_chunk_filtering.out

```diff
@@ -1141,4 +1143,21 @@ SELECT * FROM pushdown_test WHERE country IN ('USA', 'ZW', volatileFunction());
   8 | ZW
 (3 rows)
 
-DROP TABLE pushdown_test;
+DROP SCHEMA columnar_chunk_filtering CASCADE;
```

Comment:
let also do `set client_min_messages to warning` or such before this line because the order of `drop cascades ..` messages might be flaky later